test/functorch/test_aotdispatch.py

```diff
@@ -1122,9 +1122,10 @@ def forward(self, primals_1):
 
     def test_input_mutation_storage_resize_down(self):
         def f(a):
-            out = a.sin()
-            torch.ops.inductor.resize_storage_bytes_(a, 0)
-            return out
+            with torch.no_grad():
+                out = a.sin()
+                torch.ops.inductor.resize_storage_bytes_(a, 0)
+                return out
```

Comment:
The original joint graph looks like:
```
a.sin()
torch.ops.inductor.resize_storage_bytes_(a, 0)
a.cos()
```
and is not a valid graph (running it will segfault because we are doing `a.cos()` after resizing `a` to 0).
Changing it to inference only to hopefully still captures the gist of the test.